dashboard/config/routes.rb

```diff
@@ -768,8 +768,10 @@
       concerns :section_api_routes
       post 'users/:user_id/using_text_mode', to: 'users#post_using_text_mode'
       post 'users/:user_id/display_theme', to: 'users#update_display_theme'
+      post 'users/:user_id/mute_music', to: 'users#post_mute_music'
```

Comment:
For my knowledge: does this just work by some Rails magic? I don't see a controller action defined?

Reply:
🤦🏻‍♀️🤦🏻‍♀️🤦🏻‍♀️ Fixed and updated!